lte/gateway/c/session_manager/SessionState.h

```diff
@@ -167,6 +169,8 @@ class SessionState {
 
   std::vector<SetGroupPDR>& get_all_pdr_rules();
 
+  /* method to reset retransmit  count */
+  void reset_rtx_counter();
```

Comment:
here too, space between the two functions

Reply:
Space has been added b/w two function declaration.